### DISCUSSION TITLE
Finetuning does not work despite successfully loading pretrained model.

### DISCUSSION THREAD

**Discussion by sreid19** [PaddlePaddle/PaddleOCR#12552] [Q&A]
Hello,

I am finetuning en_PP-OCRv4_rec_train. The inference model gives around 90-95% accuracy on my dataset, I was hoping to do finetuning to sort out some minor issue recognizing the difference between D and O. However, when I run finetuning, the model accuracy starts from 0. 

I know the pretrained model is correctly loading: 

load pretrain successful from C:/Users/scott/anaconda3/envs/paddleocr/Lib/site-packages/PaddleOCR/pretrain_model/en_PP-OCRv4_rec_train/best_accuracy

I downloaded the training files directly from this Git and un-tarred it using 7zip. The model accuracy does climb throughout training and the evaluation accuracy eventually gets to around 75%. This is worse than the standard PP-OCRv4 inference model. I really would like to just finetune the model instead of retraining the whole thing. Any ideas?
